Micromouse/PIDController.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+class PIDController {
+  public:
+    class Delta {
+      public:
+        Delta(){};
+        virtual unsigned operator()() const {
```

Comment:
Is there any case when this default implementation should be used? If not, it should be a pure virtual function.